alfasim_sdk/types.py

```diff
@@ -89,15 +89,44 @@ def check(self, attr: Attribute, values: List[str]) -> None: # pylint: disable=a
 
 
 @attr.s(kw_only=True)
-class DataReference(BaseField):
-    value = attrib()
+class Reference(BaseField):
+    ref_type = attrib()
 
-    @value.validator
+    @ref_type.validator
     def check(self, attr: Attribute, value: Type[ALFAsimType]) -> None:
         if not issubclass(value, ALFAsimType):
             raise TypeError(f"{attr.name} must be a valid ALFAsim type")
 
 
+@attr.s(kw_only=True)
+class ReferenceSelection(BaseField):
+    """
+    ReferenceSelection allows the user to select multiples models as a "checklist" component.
+    This components only lists models that are Container Models "alfasim_sdk.models.container_models"
+
+    Properties:
+        caption   - property used as a label for the text input.
+        list_type - property that holds the container_model selected.
+
+    """
+    list_type = attrib()
```

Comment:
@nicoddemus @tarcisioe And how bout about this name?

We call `list_type`,  but we actually just accept a single class that it's decorated by `@container_model`

Reply:
`container_type`?